python/paddle/fluid/backward.py

```diff
@@ -1554,12 +1581,32 @@ def _find_no_grad_vars(block, op_path, targets, no_grad_set):
     return set(no_grad_var)
 
 
-def _find_op_path_(block, outputs, inputs, no_grad_set):
+def _find_op_path_(block,
+                   targets,
+                   inputs,
```

Comment:
Not important. why modify `outputs` into `targets`? It seems `targets` is more clear. please modify the `outputs(list[Variable]):` in `Args` into `targets(list[Variable]):`

Reply:
Thanks, copy error, done